src/core/ht.c

```diff
@@ -27,6 +27,10 @@ static ht_data_t ht_data;
 static const uint8_t frame_period = 10;
 static const uint8_t sync_len = 200;
 
+static bool has_motion_data = false;
+static int is_moving;
```

Comment:
can we make this a `bool`? its exclusively used as one.
dito in as `detect_motion` parameter